Library/Homebrew/utils/repology.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "utils/curl"
+require "utils/versions"
+
+require "formula_info"
+
+module RepologyParser
+  module_function
+
+  MAX_PAGINATION = 15
+
+  def query_api(last_package_in_response = "")
+    last_package_in_response += "/" unless last_package_in_response.empty?
+
+    url = "https://repology.org/api/v1/projects/#{last_package_in_response}?inrepo=homebrew&outdated=1"
+    ohai "Calling API #{url}" if Homebrew.args.verbose?
+
+    output, _errors, _status = curl_output(url.to_s)
+    JSON.parse(output)
+  end
+
+  def parse_api_response
+    ohai "Querying outdated packages from Repology"
+
+    outdated_packages = query_api
+    last_package_index = outdated_packages.size - 1
+    response_size = outdated_packages.size
+    page_no = 1
+
+    while response_size > 1 && page_no <= MAX_PAGINATION
+      ohai "Paginating Repology api page: #{page_no}" if Homebrew.args.verbose?
```

Comment:
```suggestion
      odebug "Paginating Repology API page: #{page_no}"
```

I feel this is more of a "debug" message than a "verbose" message.